lua/telescope/pickers/entry_display.lua

```diff
@@ -43,8 +43,8 @@ end
 
 local function truncate(str, len)
   -- TODO: This doesn't handle multi byte chars...
-  if vim.fn.strdisplaywidth(str) > len - 1 then
-    str = str:sub(1, len)
+  if vim.fn.strdisplaywidth(str) > len then
+    str = str:sub(1, len - 1)
```

Comment:
We also have to subtract 1 here because if we truncate the string is actually 1 to long, so the separator is shifted 1 to the right

Reply:
Yeah, that makes sense as well.